scorecard/templates/profile/_performance.html

```diff
@@ -1,736 +1,863 @@
-				</td>
-			    </tr>
-			    <tr class="table-footer">
-				<td>
-				    {% if opinion.report_url %}
-				    <a href="{{ opinion.report_url }}" target="_blank">Read report <i class="fa fa-file-pdf-o" aria-hidden="true"></i></a>
-				    {% else %}
-				    No report available
-				    {% endif %}
-				</td>
-			    </tr>
-			</table>
-		    </div>
-		    {% endfor %}
-		</div>
 
-		<div class="calc">
-		    <div><span class="h5">Source:</span> <a href="http://mfma.treasury.gov.za/Documents/07.%20Audit%20Reports" target="_blank">Municipal Audit Reports</a></div>
+			<div class="col-sm-8">
+				<div class="row">
+					{% for opinion in audit_opinions.values %}
+					<div class="col-sm-3">
+						<table class="table table-bordered">
+							<tr>
+								<td class="text-center"><strong>
+										{{ opinion.date|unlocalize }}
+										<i class="audit-rating audit-{{ opinion.rating }}" aria-hidden="true"></i>
+										{% if opinion.rating == "unqualified" %}
+										<i class="audit-rating audit-unqualified_extra" area-hidden="true"></i>
+										{% endif %}
+										<br>{{ opinion.result }}
+									</strong>
+								</td>
+							</tr>
+							<tr class="table-footer">
+								<td>
+									{% if opinion.report_url %}
+									<a href="{{ opinion.report_url }}" target="_blank">Read report <i
+											class="fa fa-file-pdf-o" aria-hidden="true"></i></a>
+									{% else %}
+									No report available
+									{% endif %}
+								</td>
+							</tr>
+						</table>
+					</div>
+					{% endfor %}
+				</div>
+
+				<div class="calc">
+					<div><span class="h5">Source:</span> <a
+							href="http://mfma.treasury.gov.za/Documents/07.%20Audit%20Reports" target="_blank">Municipal
+							Audit Reports</a></div>
+				</div>
+			</div>
+
+			<div class="col-sm-4">
+				<div class="panel panel-didyouknow">
+					<div class="panel-heading"><i class="fa fa-info-circle" aria-hidden="true"></i> Did you know?</div>
+					<div class="panel-body">
+						<p>There are <strong>5 types of audit outcomes.</strong></p>
+						<div class="more">
+							<a href="#more-audit-opinions" data-toggle="collapse"
+								class="show-more-audit-opinions show-more"><span class="icon"><i class="fa fa-plus"
+										aria-hidden="true"></i></span> Learn about audit outcomes</a>
+						</div>
+					</div>
+				</div>
+			</div>
+
 		</div>
-	    </div>
-
-	    <div class="col-sm-4">
-		<div class="panel panel-didyouknow">
-		    <div class="panel-heading"><i class="fa fa-info-circle" aria-hidden="true"></i> Did you know?</div>
-		    <div class="panel-body">
-			<p>There are <strong>5 types of audit outcomes.</strong></p>
-			<div class="more">
-			    <a href="#more-audit-opinions" data-toggle="collapse" class="show-more-audit-opinions show-more"><span class="icon"><i class="fa fa-plus" aria-hidden="true"></i></span> Learn about audit outcomes</a>
+		<div class="more-audit-opinions collapse panel panel-didyouknow" id="more-audit-opinions">
```

Comment:
tabs :(

Reply:
@jbothma Looks like there was a mix of tabs and spaces before my editor got involved. I've turned off my HTML auto formatting for now and included only the required changes.